src/sif/pedestriancost.cc

```diff
@@ -257,17 +257,6 @@ class PedestrianCost : public DynamicCost {
     return true;
   }
 
-  /**
-   * This method overrides the max_distance with the max_distance_mm per segment
-   * distance. An example is a pure walking route may have a max distance of
-   * 10000 meters (10km) but for a multi-modal route a lower limit of 5000
-   * meters per segment (e.g. from origin to a transit stop or from the last
-   * transit stop to the destination).
-   */
-  virtual void UseMaxMultiModalDistance() override {
```

Comment:
I think we need this as we would fail vs just walking.  We saw that we would prefer long walking routes vs taking transit.  

Also, I think we did this for transfers from one line to another.  Think about taking the PATH and walking for awhile then taking the MTA in NYC.  At that time they were separate operators/agencies.  Not sure if the feed is like that now as MTA owns PATH now, but need to test that case again.  It would be more desirable to just transfer at a station that has stops for both operators/agencies.